null_safety_examples/misc/lib/tutorial/stream_interface.dart

```diff
@@ -1,7 +1,7 @@
 // ignore_for_file: annotate_overrides, type_annotate_public_apis, strict_raw_type, use_function_type_syntax_for_parameters
 import 'dart:async';
 
-// TODO(chalin): I believe that handleError's test argument should be declared
+// TODO(chalin): I believe that handleError's test argument should be declared (#3298)
 // as follows, but it isn't: {bool Function(dynamic error) test}.
```

Comment:
Can you instead put a link to the issue so its easier to access when navigating the source?

```suggestion
// TODO(chalin): I believe that handleError's test argument should be declared
// as follows, but it isn't: {bool Function(dynamic error) test}.
// (https://github.com/dart-lang/site-www/issues/3298)
```